cheatsheets/DotNet_Security_Cheat_Sheet.md

```diff
@@ -609,17 +613,63 @@ public void RemoveAntiForgeryCookie(Controller controller)
 }
 ```
 
-NB: You will need to attach the anti-forgery token to Ajax requests.
+#### Using .NET Core 2.0 or later:
+
+After .NET Core 2.0 it is possible to [automatically generate and verify the antiforgery token](https://github.com/aspnet/AspNetCore.Docs/blob/master/aspnetcore/security/anti-request-forgery.md#aspnet-core-antiforgery-configuration).
```

Comment:
`After` here is ambiguous. You might mean `Starting with`...

Reply:
Thanks for the feedback, I learned something as a non-native speaker ;) I changed it.